controllers/help.js

```diff
@@ -4,57 +4,213 @@ const router = express.Router();
 const authHelper = require('../helpers/authentication');
 const permissionHelper = require('../helpers/permissions');
 const api = require('../api');
-const showdown  = require('showdown');
+const showdown = require('showdown');
 const converter = new showdown.Converter();
 const moment = require('moment');
 
 const faq = require('../helpers/content/faq.json');
 
+/*
+
+// GET tutorials.json
+
+async function articles(){
+    const cardsUrl = `https://docs.schul-cloud.org/rest/api/content/13828239/child/page?expand=children.page`;
+    const cardsResult = await fetch(cardsUrl);
+    const cardJson = await cardsResult.json();
+    let cards = cardJson.results.map((category) => {
+        return {
+            id: category.id,
+            title: category.title,
+            type: "topic"
+        };
+    }).map(async (card) => {
+        const articleResult = await fetch(`https://docs.schul-cloud.org/rest/api/content/${card.id}/child/page?expand=children.page`);
+        const articleJson = await articleResult.json();
+        let categories = articleJson.results.map((category) => {
+            return {
+                id: category.id,
+                title: category.title,
+                type: "category",
+                articles: category.children.page.results.map((article) => {
+                    return {
+                        id: article.id,
+                        title: article.title,
+                        type: "article"
+                    };
+                })
+            };
+        });
+        card.categories = categories;
+        return card;
+    });
+    let content = await Promise.all(cards);
+    let w = window.open();
+    w.document.write(JSON.stringify(content));
+}
+articles();
+*/
```

Comment:
Ist das absichtlich auskommentiert und nicht gelöscht?

Reply:
ja, das Skript wird benötigt um die tutorials.json zu erstellen. Soll sobald das hier erstmal durch ist in Phase 2 in den Server + SuperHero Dashboard verlagert werden.